doc/source/api/sequence_annotations.rst

```diff
@@ -0,0 +1,36 @@
+.. _sequence_annotations:
+
+************************
+Sequence Annotations API
+************************
+For the Sequence Annotation schema definitions, see `Sequence Annotation schema <../schemas/sequenceAnnotations.html>`_
+
+
+------------------------
+Feature Based Hierarchy
+------------------------
+The central object of the GA4GH Sequence Annotation API is a Feature.  The Feature describes an interval of interest on some reference(s).  It has a span from a start position to a stop position as well as descriptive data.  A Feature has one or more parent Features which enables the construction of more complex representations in a hierarchical way.
```

Comment:
Whoops, that crept in by accident. Should have zero or one parents in current schema. Thanks!